src/modal/test/modal.spec.js

```diff
@@ -1064,12 +1064,12 @@ describe('$uibModal', function() {
 
     describe('controller', function() {
       it('should accept controllers and inject modal instances', function() {
-        var TestCtrl = function($scope, $uibModalInstance) {
```

Comment:
I don't understand your comment, this test is for the default value, and there is another test to override this value.
Before this test use the $scope but if we add a default value the $scope variables will not work anymore.